packages/server/package.json

```diff
@@ -115,6 +115,7 @@
     "trash": "5.2.0",
     "tree-kill": "1.2.2",
     "ts-node": "8.5.4",
+    "type-is": "1.6.18",
```

Comment:
I checked `yarn.lock` and `type-is` existed. So, it doesn't increase the bundle size.

Reply:
Rolled back because of unit test failure.